src/content/glossary/index.md

```diff
@@ -173,19 +172,19 @@ Converting code written in a high-level programming language (e.g., [Solidity](#
 
 ### committee {#committee}
 
-A group of at least 128 [validators](#validator) assigned to validate blocks in each slot. One of the validators in the committee is the aggregator, responsible for aggregating the signatures of all other validators in the committee that agree on an attestation. Not to be confused with [sync committee](#sync-committee).
+A group of at least 128 [validators](#validator) assigned to validate blocks in each slot. One of the validators in the committee is the aggregator, responsible for aggregating the signatures of all other validators in the committee that agree on an attestation. Committees rotate such that every active validator attests in every [epoch](#epoch). Not to be confused with [sync committee](#sync-committee).
 
 ### computational infeasibility {#computational-infeasibility}
 
 A process is computationally infeasible if it would take an impracticably long time (e.g. billions of years) to do it for anyone who might conceivably have an interest in carrying it out.
 
 ### consensus {#consensus}
 
-When a supermajority of nodes on the network all have the same blocks in their locally validated best blockchain. Not to be confused with [consensus rules](#consensus-rules).
+When a supermajority of nodes on the network all have the same blocks in their local copy of the blockchain. Not to be confused with [consensus rules](#consensus-rules).
```

Comment:
what is a supermajority? can we explain better?

Reply:
```suggestion
When a supermajority (more than 66%) of nodes on the network all have the same blocks in their local copy of the blockchain. Not to be confused with [consensus rules](#consensus-rules).
```

@konopkja adding a change that should address this, let me know what you think